src/redux/book/book.js

```diff
@@ -0,0 +1,26 @@
+import { createAction } from '@reduxjs/toolkit/src/createAction';
+import { nanoid } from '@reduxjs/toolkit';
+
+const INITIAL_STATE = [];
+
+const bookReducer = (state = INITIAL_STATE, actions) => {
+  if (actions.type === 'ADD_BOOK') {
+    return [...state, actions.payload];
+  } if (actions.type === 'REMOVE_BOOK') {
+    return [state.filter((book) => book !== actions.payload.id)];
+  }
```

Comment:
- [ ] Please make sure that you define a variable `ADD_BOOK` and `REMOVE_BOOK` as action types and the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. :pray: